lib/jxl/enc_ac_strategy.cc

```diff
@@ -492,56 +492,56 @@ uint8_t FindBest8x8Transform(size_t x, size_t y, const ACSConfig& config,
     float entropy_add;
     float entropy_mul;
   };
-  const TransformTry8x8 kTransforms8x8[] = {
+  static const TransformTry8x8 kTransforms8x8[] = {
       {
           AcStrategy::Type::DCT,
-          0.0f,
-          0.80f,
+          3.0f,
+          0.8425438883708443f,
```

Comment:
Perhaps a comment about where these numbers are coming from (how were they tuned?) would be good.

Is 'best' 8x8 transform here something referring to lowest estimated entropy (so best for density), or referring to 'best overall choice for quality/bpp'? This is not really clear to me. I assume it's the latter.

Also: these constants are the same for the whole spectrum of distance targets. Does that make sense?

Reply:
added a comment in the very beginning